suitcase/tests/test_spec.py

```diff
@@ -205,6 +205,12 @@ def test_round_trip_from_run_engine(db_all):
 
     #with pytest.raises(NotImplementedError):
     sf = spec.Specfile(fname)
+    sf1 = _round_trip(sf, db_all)
+
+    # a2scan is not round trippable
```

Comment:
Hmm. Why?

Reply:
Old comment, not from me. We may need to update it.